_episodes/01-introduction.md

```diff
@@ -11,5 +11,121 @@ keypoints:
 ---
 The example-based nature of [Matplotlib documentation](https://matplotlib.org/) is GREAT.
 
-{% include links.md %}
+Matplotlib is the standard when it comes to making plots in Python. It is versatile and allows for lots of functionality and different ways to produce many plots.
+We will be focusing on using matplotlib for High Energy Physics.
+
+# A simple example
+A with any python code it is always a good practice to import the necessary libraries as a first step.
```

Comment:
Python should be written with capital P (that is what the official docs say).

Reply:
I am not very familiar with Github (we use Bitbucket at Belle II), but I think there is a way to just change directly on the page. In any case, a change directly in the page will create a commit, then it is the same.